x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/kube_list_action.tsx

```diff
@@ -0,0 +1,260 @@
+  }
+
+  const diffInSeconds = moment(moment.now()).diff(moment(startTime), 'seconds');
+  const remainingSeconds = diffInSeconds - diffInMinutes * 60;
+
+  if (diffInMinutes > 0) {
+    return `${diffInMinutes}m${remainingSeconds > 0 ? `${remainingSeconds}s` : ''}`;
+  }
+
+  return `${diffInSeconds}s`;
+};
+
+// @ts-expect-error TS2769
+const StyledEuiBasicTable = styled(EuiBasicTable)`
+  table {
+    background-color: ${({ theme: { eui } }) => eui.euiPageBackgroundColor};
+  }
+  .euiTableHeaderCell {
+    border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+    .euiTableCellContent__text {
+      font-weight: ${(props) => props.theme.eui.euiFontWeightRegular};
+    }
+  }
+  .euiTableRow {
+    &:hover {
+      background-color: white !important;
+    }
+    .euiTableRowCell {
+      border-top: none !important;
+      border-bottom: none !important;
+    }
+  }
+`;
+
+const COLUMNS_TO_RESOURCE_MAPPING = {
+  pod: [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.name',
+        { defaultMessage: 'NAME' }
+      ),
+      width: '20%',
+    },
+    {
+      field: 'ready',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.ready',
+        { defaultMessage: 'READY' }
+      ),
+      width: '10%',
+    },
+    {
+      field: 'status',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.status',
+        { defaultMessage: 'STATUS' }
+      ),
+      width: '10%',
+    },
+    {
+      field: 'restarts',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.restarts',
+        { defaultMessage: 'RESTARTS' }
+      ),
+      width: '10%',
+    },
+    {
+      field: 'age',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.age',
+        { defaultMessage: 'AGE' }
+      ),
+      width: '10%',
+    },
+    {
+      field: 'ip',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.ip',
+        { defaultMessage: 'IP' }
+      ),
+      width: '15%',
+    },
+    {
+      field: 'node',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.node',
+        { defaultMessage: 'NODE' }
+      ),
+      width: '25%',
+    },
+  ],
+  deployment: [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.name',
+        { defaultMessage: 'NAME' }
+      ),
+      width: '20%',
+    },
+    {
+      field: 'ready',
+      name: i18n.translate(
+        'xpack.securitySolution.endpointResponseActions.getKubeList.table.header.ready',
+        { defaultMessage: 'READY' }
+      ),
```

Comment:
The first two columns are the same in both tables. Consider saving them into variables and using those. You could also use a function expression to create the first two columns and then use that in the `COLUMNS_TO_RESOURCE_MAPPING` list